rfcs/text/0005/ec2.yml

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
Can this example file be renamed from `ec2.yml` to `ec2.json`?

Reply:
Done, thank you for the review!